--- ./platform_linux/build.sh	2018-07-09 15:57:57.668782630 -0700
+++ ./platform_linux/build.sh	2018-07-09 16:00:14.483328154 -0700
@@ -35,7 +35,7 @@
 
 case $OS in
     Ubuntu|Debian|DebianJessie32bit)
-        echo "apt-get install -y libtool libudev-dev automake autoconf ant curl lib32z1 lib32ncurses5 lib32bz2-1.0 p7zip-full"
+        echo "apt-get install -y libtool libudev-dev automake autoconf ant curl lib32z1 lib32ncurses5 libbz2-1.0 p7zip-full"
       if [ $OS==DebianJessie32bit ]; then
             sudo apt-get install -y libtool libudev-dev automake autoconf \
                ant curl p7zip-full fakeroot
@@ -47,7 +47,7 @@
         # On more recent versions of Ubuntu
         # the libbz2 package is multi-arch
         install_lib_bz2() {
-            sudo apt-get install -y lib32bz2-1.0
+            sudo apt-get install -y libbz2-1.0
         }
         set +e
         if ! install_lib_bz2; then
@@ -188,8 +188,8 @@
 
 case $OS in
     Ubuntu|Debian)
-        echo "apt-get install openjdk-7-jdk"
-        sudo apt-get install openjdk-7-jdk
+        echo "apt-get install openjdk-8-jdk"
+        sudo apt-get install openjdk-8-jdk
         ;;
     Archlinux)
         echo "pacman -Syy jdk7-openjdk"
