--- ./platform_linux/build.sh	2019-02-23 11:20:37.944042668 -0800
+++ ./platform_linux/build.sh	2019-02-23 11:24:14.600854324 -0800
@@ -35,7 +35,7 @@
 
 case $OS in
     Ubuntu|Debian|DebianJessie32bit)
-        echo "apt-get install -y libtool libudev-dev automake autoconf ant curl lib32z1 lib32ncurses5 lib32bz2-1.0 p7zip-full"
+        echo "apt-get install -y libtool libudev-dev automake autoconf ant curl lib32z1 lib32ncurses5 libbz2-1.0 p7zip-full"
       if [ $OS==DebianJessie32bit ]; then
             sudo apt-get install -y build-essential libtool libudev-dev automake autoconf \
                ant curl p7zip-full fakeroot unzip udev
@@ -47,7 +47,7 @@
         # On more recent versions of Ubuntu
         # the libbz2 package is multi-arch
         install_lib_bz2() {
-            sudo apt-get install -y lib32bz2-1.0
+            sudo apt-get install -y libbz2-1.0
         }
         set +e
         if ! install_lib_bz2; then
